openjdk9/src/test/resources/openjdk9-structure.yaml

```diff
@@ -0,0 +1,13 @@
+# Test spec for the Cloud Runtimes container structure testing framework.
+#
+# See https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/README.md for more detail.
+#
+schemaVersion: 1.0.0
+
+commandTests:
+- name: 'correct java version is installed'
+  command: ['java', '-version']
+  expectedError: ['openjdk version "${openjdk.version.major}-Debian"']
```

Comment:
The `-Debian` suffix likely is there due to the fact that we're installing openjdk currently from the "unstable" debian package repository. I'm not sure if debian builds of openjdk9 will always have this suffix, but I'd hope at least that they will follow the new version scheme spec. I haven't seen clear info on this yet.